docs/12-upgrading/01-upgrade-to-two.md

```diff
@@ -212,4 +212,52 @@ The authentication module's default value for allowing unsecure random number ge
 auth.AuthConfig.set(auth.AuthConfig(
   allowUnsecureRandom: true,
 ));
+```
+
+## Updates to Serialization in Serverpod 2.0
+
+### General Changes to Model Serialization
+Serverpod 2.0 introduces a significant simplification in the serialization process of all models. Previously, the `fromJson` factory constructors required a `serializationManager` parameter to deserialize objects. This parameter is no longer needed, streamlining the deserialization process for all models, not just custom ones.
```

Comment:
The intention for the end of this line is repeated on line 236. Please remove it from here and only mention custom models down there.